core/lib/contracts/src/lib.rs

```diff
@@ -393,3 +393,314 @@ impl BaseSystemContracts {
         }
     }
 }
+
+pub static COMMIT_BLOCKS_FUNCTION: Lazy<Function> = Lazy::new(|| {
```

Comment:
Maybe call it `PRE_BOOJUM_COMMIT_BLOCKS_FUNCTION`?

Reply:
`BLOCKS` in name implicitly states it's pre-boojum as for post-boojum we have `BATCHES`, but I agree that `PRE_BOOJUM` prefix would look better. I will change it to `PRE_BOOJUM_COMMIT_FUNCTION`